### PR TITLE
refactor: remove unused hooks

### PR DESCRIPTION
**Pull request by Himself65**

### CODE REVIEW

File: src/common/components/Translator.tsx [hunk 1]
```diff
@@ -547,16 +547,6 @@ export function Translator(props: IPopupCardProps) {
     const stopLoading = useCallback(() => {
         setIsLoading(false)
     }, [])
-    useEffect(() => {
-        setOriginalText(props.text)
-    }, [props.text, props.uuid])
-    useEffect(() => {
-        setEditableText(originalText)
-        setSelectedWord('')
-    }, [originalText, props.uuid])
```
Comment (yetone) [inline src/common/components/Translator.tsx:556]: If this is deleted, the changes in the original text will not be synced to the editable text

Comment (yuchanns) [reply src/common/components/Translator.tsx:556]: It seems that's why he passes the key `${uuid}-${text}` to the `<Translator>` component, which will trigger re-render.

Comment (Himself65) [reply src/common/components/Translator.tsx:556]: Please read this article

https://react.dev/learn/you-might-not-need-an-effect#resetting-all-state-when-a-prop-changes

Comment (yetone) [reply src/common/components/Translator.tsx:556]: @Himself65 Great article, but will this leak component internal logic to the user? Will it increase the mental burden of the user?

**Pull request merged**